otherlibs/stdlib_stable/int.ml

```diff
@@ -26,87 +26,61 @@ let size = Sys.int_size
 external ( < ) : int -> int -> bool = "%int_lessthan"
 
 let zero = 0
-
 let one = 1
-
 let minus_one = -1
 
 external to_int : int -> int = "%identity"
-
 external of_int : int -> int = "%identity"
-
 external neg : int -> int = "%int_neg"
-
 external add : int -> int -> int = "%int_add"
-
 external sub : int -> int -> int = "%int_sub"
-
 external mul : int -> int -> int = "%int_mul"
-
 external div : int -> int -> int = "%int_div"
-
 external rem : int -> int -> int = "%int_mod"
-
 external succ : int -> int = "%int_succ"
-
 external pred : int -> int = "%int_pred"
-
 external logand : int -> int -> int = "%int_and"
-
 external logor : int -> int -> int = "%int_or"
-
 external logxor : int -> int -> int = "%int_xor"
 
 let[@inline] lognot x = logxor x minus_one
 
 external shift_left : int -> int -> int = "%int_lsl"
-
 external shift_right : int -> int -> int = "%int_asr"
-
 external shift_right_logical : int -> int -> int = "%int_lsr"
 
 let[@inline] abs x = if x < zero then neg x else x
 
 external equal : int -> int -> bool = "%int_equal"
-
 external compare : int -> int -> int = "%int_compare"
 
 let[@inline] min x y = if x < y then x else y
-
 let[@inline] max x y = if x < y then y else x
 
 external of_float : float -> int = "%int_of_float"
-
 external to_float : int -> float = "%float_of_int"
-
 external format_int : string -> int -> string = "caml_format_int"
 
 let[@inline] to_string t = format_int "%d" t
 
 external of_string : string -> int = "caml_int_of_string"
 
 let max_int = shift_right_logical minus_one 1
-
 let min_int = succ max_int
-
 let[@inline] unsigned_to_int t = if t < 0 then None else Some t
 
-let[@inline] unsigned_compare n m = compare (sub n min_int) (sub m min_int)
-
-let[@inline] unsigned_lt n m = sub n min_int < sub m min_int
+external unsigned_compare : int -> int -> int = "%int_unsigned_compare"
+external unsigned_lt : int -> int -> bool = "%int_unsigned_lessthan"
```

Comment:
I didn't expect code changes in a PR moving things to stable - what's going on in this case?